pyro/distributions/conditional.py

```diff
@@ -0,0 +1,63 @@
+from abc import abstractmethod
+
+import torch
+import torch.nn
+
+
+class ConditionalDistribution(object):
```

Comment:
I think you'll want to inherit from `ABC` rather than `object`, here and in all other abstract classes
```py
from abc import ABC, abstractmethod

class ConditionalDistribution(ABC):
    ...
```

Reply:
Fixed!